File: poh20/cpp/5.cpp
//usr/bin/env true; tmpfile=$(mktemp); g++ -O2 -std=gnu++11 -xc++ -o $tmpfile $0 && $tmpfile "$@"; rm $tmpfile; exit
#import<iostream>
#import<vector>
#import<numeric>
using namespace std;
int main(){
int n;cin>>n;
vector<int>v;v.resize(n);
iota(v.begin(),v.end(),1);
cout<<accumulate(v.begin(),v.end(),0)<<endl;}

File: poh20/rust/5.rs
//usr/bin/env true; tmpfile=$(mktemp); rustc -o $tmpfile $0 && $tmpfile "$@"; rm $tmpfile; exit
fn l()->String{let mut x=String::new();std::io::stdin().read_line(&mut x).ok();return x;}
fn main(){println!("{}",(1..=l().trim().parse().unwrap()).sum::<i32>());}
